cluster/get-kube.sh

```diff
@@ -120,7 +120,7 @@ if [[ -n "${KUBERNETES_SKIP_CONFIRM-}" ]]; then
 fi
 
 if [[ $(which wget) ]]; then
-  wget -O ${file} ${release_url}
+  wget -N ${release_url}
 elif [[ $(which curl) ]]; then
   curl -L -o ${file} ${release_url}
```

Comment:
Can you also fix the case for curl users? As @chrisleck mentioned in #15197 this should be possible with the `-z` flag.